lib/network.sh

```diff
@@ -115,7 +110,50 @@ if [[ "${EPHEMERAL_CLUSTER}" == "minikube" ]] && [[ -n "${EXTERNAL_SUBNET_V6}" ]
     network_address MINIKUBE_BMNET_V6_IP "${EXTERNAL_SUBNET_V6}" 9
 fi
 
-export REGISTRY=${REGISTRY:-"${EXTERNAL_SUBNET_V4_HOST}:5000"}
+
+if [[ -n "${EXTERNAL_SUBNET_V4}" ]]; then
+  prefixlen EXTERNAL_SUBNET_V4_PREFIX "$EXTERNAL_SUBNET_V4"
+  export EXTERNAL_SUBNET_V4_PREFIX
+  if [[ -z "${EXTERNAL_SUBNET_V4_HOST:-}" ]]; then
+    network_address EXTERNAL_SUBNET_V4_HOST "$EXTERNAL_SUBNET_V4" 1
+  fi
+
+  # Calculate DHCP range for baremetal network (20 to 60 is the libvirt dhcp)
+  network_address BAREMETALV4_POOL_RANGE_START "$EXTERNAL_SUBNET_V4" 100
+  network_address BAREMETALV4_POOL_RANGE_END "$EXTERNAL_SUBNET_V4" 200
+  export BAREMETALV4_POOL_RANGE_START
+  export BAREMETALV4_POOL_RANGE_END
+else
+  export EXTERNAL_SUBNET_V4_PREFIX=""
+  export EXTERNAL_SUBNET_V4_HOST=""
+  export BAREMETALV4_POOL_RANGE_START=""
+  export BAREMETALV4_POOL_RANGE_END=""
+fi
+
+if [[ -n "${EXTERNAL_SUBNET_V6}" ]]; then
+  prefixlen EXTERNAL_SUBNET_V6_PREFIX "$EXTERNAL_SUBNET_V6"
+  export EXTERNAL_SUBNET_V6_PREFIX
+  if [[ -z "${EXTERNAL_SUBNET_V6_HOST}" ]]; then
+    network_address EXTERNAL_SUBNET_V6_HOST "$EXTERNAL_SUBNET_V6" 1
+  fi
+
+  # Calculate DHCP range for baremetal network (20 to 60 is the libvirt dhcp) IPv6
+  network_address BAREMETALV6_POOL_RANGE_START "$EXTERNAL_SUBNET_V6" 100
+  network_address BAREMETALV6_POOL_RANGE_END "$EXTERNAL_SUBNET_V6" 200
+  export BAREMETALV6_POOL_RANGE_START
+  export BAREMETALV6_POOL_RANGE_END
```

Comment:
After network_address or prefixlen are called, the variables are already exported. Some exports may be unnecessary here